### ISSUE TITLE
Error from ApiStatusCodesTable

### ISSUE THREAD

**Issue by pcapanna** [cloud-annotations/docusaurus-plugin-openapi#19]
Im using this as dependency for another project and tried updating to v0.0.13 but encontered an error when trying to display the API. 

It doesn´t ever show the page. I tried downloading this example and compile it myself but no changes. Even the example seems broken to me. I don´t know why you guys are not having the same error as me.. I tracked the error back to commit 193710b088a97fd7bf5f5641e51d875fc26e340b (10/12/2020) 
I got an easy fix but I didn´t explore it deeply. 

It seems that from that commit you are using ApiRequestBodyTable to display ApiStatusCodeTable. But at ApiStatusCodes the body not always provides a content.. And since from ApiRequestTableBodyTable we have this code:

```
  if (body === undefined) {
    return null;
  }

  // NOTE: We just pick a random content-type.
  // How common is it to have multiple?

  const randomFirstKey = Object.keys(body.content)[0];
```

It fails to access 0 index of undefined.

Easy fix: 
```
  if (body?.content === undefined) {
    return null;
  }
  const randomFirstKey = Object.keys(body.content)[0];
```

I will make a pull request if it seams fair.

**Comment by bourdakos1** [comment]
Oh yea I think this error was introduced in the PR we just merged

**Comment by bourdakos1** [comment]
You fix looks good to me, happy to merge a PR if you open one

**Comment by bourdakos1** [comment]
Should be fixed I v0.0.14
